dts/bindings/gpio/nordic,nrf-gpiote.yaml

```diff
@@ -5,7 +5,7 @@
 #
 ---
 title: NRF5 GPIOTE
```

Comment:
NRF5 -> nRF
Same applies to [line 12](https://github.com/zephyrproject-rtos/zephyr/pull/8729/files#diff-421ee7681169f4fe7e16f5bada858df5R12).